src/main/java/frc/robot/subsystems/Arm/ArmSubsystem.java

```diff
@@ -0,0 +1,340 @@
+package frc.robot.subsystems.Arm;
+
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.Extendo.ExtendoSubsystem;
+import frc.robot.subsystems.RobotState.RobotStateSubsystem;
+import frc.robot.subsystems.RobotState.RobotStateSubsystem.gamePiece;
+import frc.robot.subsystems.Wrist.WristSubsystem;
+import frc.robot.subsystems.shoulder.ShoulderSubsystem;
+import java.util.Set;
+
+import org.strykeforce.telemetry.measurable.MeasurableSubsystem;
+import org.strykeforce.telemetry.measurable.Measure;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ArmSubsystem extends MeasurableSubsystem {
+
+  private final ShoulderSubsystem shoulderSubsystem;
+  private final ExtendoSubsystem extendoSubsystem;
+  private final WristSubsystem wristSubsystem;
+  private org.slf4j.Logger logger = LoggerFactory.getLogger(ArmSubsystem.class);
+  private ArmState currState;
+  private ArmState desiredState;
+  private ArmState finalState;
+  private boolean useFinal;
+
+  public ArmSubsystem(
+      ShoulderSubsystem shoulderSubsystem,
+      ExtendoSubsystem extendoSubsystem,
+      WristSubsystem wristSubsystem) {
+    this.shoulderSubsystem = shoulderSubsystem;
+    this.extendoSubsystem = extendoSubsystem;
+    this.wristSubsystem = wristSubsystem;
+    currState = ArmState.STOW;
+    desiredState = ArmState.STOW;
+    finalState = ArmState.STOW;
+    useFinal = false;
+  }
+
+  public void zero() {
+    shoulderSubsystem.zero();
+    extendoSubsystem.zero();
+    wristSubsystem.zero();
+  }
+
+  public void stow() {
+    if (currState != ArmState.STOW) {
+      shoulderSubsystem.setPos(ArmConstants.kStowShoulderPos);
+      extendoSubsystem.setPos(ArmConstants.kStowExtendoPos);
+      wristSubsystem.setPos(ArmConstants.kStowWristPos);
+      currState = ArmState.PARALLEL_TRANS;
+      desiredState = ArmState.STOW;
+    }
+  }
+
+  public void high(gamePiece piece) {
```

Comment:
set useFinal to false at the beginning of each one of these methods (low, mid, high, ect) - makes sure that we don't rely on other code to reset this in case we are interrupted